marcdomain/Fast-Food-Fast
# #160644256 Ft admin post meal

### What does this PR do?
- Implementation of secured endpoint, post menu
### Description of Task to be completed
- Create post menu endpoint
- Verify user token
- Validate user inputs
- Write tests for the endpoint
### How should this be manually tested?
- Use postman to generate token via this endpoint https://marcus-fast-food-fast.herokuapp.com/api/v1/auth/login or https://marcus-fast-food-fast.herokuapp.com/api/v1/auth/signup
- Login admin using email: _admin@gmail.com_  password: _adminuser_
- Create user account that will be used to test the end point
- Use the token generated in each case to access the post-menu endpoint
- Test the implementations using postman to trigger POST on https://marcus-fast-food-fast.herokuapp.com/api/v1/menu
- Check README for the test coverage
### What are the relevant pivotal tracker stories?
- [#160644256](https://www.pivotaltracker.com/story/show/160644256)
